qemu/tests/vdpa_pxe_boot.py

```diff
@@ -0,0 +1,27 @@
+import time
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    vdpa nic PXE test:
+
+    1) Boot up guest from vdpa NIC
+    2) Check guest if works well after sleep 240s
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Try to boot from vdpa NIC", test.log.info)
+    vm = env.get_vm(params["main_vm"])
+    timeout = params.get_numeric("pxe_timeout")
+    test.log.info("Waiting %ss" % timeout)
+    time.sleep(timeout)
+    vm.verify_alive()
+    output = vm.process.get_output()
```

Comment:
This is the qemu output, not the guest serial output. So cannot be sure if the guest is booted from PXE, because there is no "Start PXE over " checkpoint.

`vm.process.get_output()` will contain other outputs.

Reply:
> This is the qemu output, not the guest serial output. So cannot be sure if the guest is booted from PXE, because there is no "Start PXE over " checkpoint.
> 
> `vm.process.get_output()` will contain other outputs.

Hi @PaulYuuu I tried to add this check, but for seabios guest , it can not got guest serial output, how should we deal with this problems?

Reply:
I have fix this problem. Since rhel9, `sgabios` output is no longer used. `machine_type_extra_params = "graphics=off" `is used instead, so I added this configuration in cfg to obtain seabios guest output.